src/components/Avatar.tsx

```diff
@@ -1,19 +1,25 @@
 'use client';
 
-import { useQuery } from '@tanstack/react-query';
-import { compact } from 'lodash-es';
 import type { ImageProps as NextImageProps } from 'next/image.js';
 import { memo, useState } from 'react';
 
 import { Image as NextImage } from '@/esm/Image.js';
 import { classNames } from '@/helpers/classNames.js';
 import { isDomainOrSubdomainOf } from '@/helpers/isDomainOrSubdomainOf.js';
-import { resolveFirstAvailableUrl } from '@/helpers/resolveFirstAvailableUrl.js';
-import { resolveImgurUrl } from '@/helpers/resolveImgurUrl.js';
 import { useIsDarkMode } from '@/hooks/useIsDarkMode.js';
 
-function resolveAvatarFallbackUrl(url: string | undefined, isDarkMode = false) {
-    if (!url?.startsWith('https://cdn.stamp.fyi/avatar/eth:')) return url;
+function resolveImgurUrl(url: string) {
+    if (!URL.canParse(url)) return;
+
+    const u = new URL(url);
+    if (u.protocol !== 'https:') return;
+    if (u.host !== 'i.imgur.com') return;
```

Comment:
Should we return original url as fallback?
```suggestion
    if (!URL.canParse(url)) return url;

    const u = new URL(url);
    if (u.protocol !== 'https:') return url;
    if (u.host !== 'i.imgur.com') return url;
```